CodeStarterCamp_Week1/Step3.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  second.swift
+//  CodeStarterCamp_Week1
+//
+//  Created by Jeong Min Lee on 2/13/24.
+//
+
+import Foundation
+
+func drawPepero(bodyWidth width: Int,pockyBody body: String ,peperoTopping topping: String, barLength length : Int) {
+    print ("""
+            <정보>
+            길이: \(width)
+            몸통: \(body)
+            토핑: \(topping)
+            막대길이: \(length)
+            """)
+    drawPeperoBody(width: width, body: body, topping: topping)
+    drawPeperoBar(length: length)
+}
+
+func drawPeperoBody(width: Int, body: String, topping: String) {
+    for _ in 1...width {
+        for _ in 1...1 {
+            print("",topping, terminator: "")
+        }
+        print(body, topping, separator: "")
```

Comment:
```suggestion
        print(topping, body, topping, separator: "")
```
`for _ in 1...1` 반복문은 반복문이라는 이름이 무색하게 무조건 1회만 실행되는 것 같습니다. 반복문을 제외하는 것이 좋겠습니다.

혹시 반복문으로 하신 이유가 있을까요?

Reply:
여태까지 반복문에 익숙해질려고 하다 보니까는 생각하지않고 반복문으로 한 것 같습니다.